app/controllers/provider_foreman_controller.rb

```diff
@@ -369,14 +374,28 @@ def tree_select_unprovisioned_configured_system
 
   def configuration_manager_providers_tree_rec
     nodes = x_node.split('-')
+    type, _id = x_node.split("_").last.split("-")
     case nodes.first
-    when "root" then find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
-    when "e"    then find_record(ManageIQ::Providers::Foreman::ConfigurationManager::ConfigurationProfile, params[:id])
-    when "cp"   then find_record(ManageIQ::Providers::Foreman::ConfigurationManager::ConfiguredSystem, params[:id])
+    when "root"    then find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
+    when "e"       then find_record(ManageIQ::Providers::Foreman::ConfigurationManager::ConfigurationProfile, params[:id])
+    when "f"       then find_record(ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfiguredSystem, params[:id])
+    when "cp"      then find_record(ManageIQ::Providers::Foreman::ConfigurationManager::ConfiguredSystem, params[:id])
```

Comment:
What are root, e, f, cp?  Can you name the `nodes.first` as a local variable to help explain what these letters mean?

Maybe not in this PR, but maybe we should create a single method that translates these ` x_node.split('-').first` into something understandable.  We seem to love to split the x_node but don't give the parts useful names so we end up comparing them to "e", "f", "cp", or something equally cryptic.